web/src/state_data.ts

```diff
@@ -139,6 +139,14 @@ export const user_group_schema = z.object({
     can_mention_group: z.number(),
 });
 
+export const user_topic_schema = z.object({
+    stream_id: z.number(),
+    topic_name: z.string(),
+    last_updated: z.number(),
+    visibility_policy: z.number(),
+    stream__name: z.string().optional(),
```

Comment:
This is preexisting, but I think this line is a bug; `stream__name` is a bit of Django database query syntax. The server-side code does this:

```
        user_topic_dict: UserTopicDict = {                                                              
            "stream_id": row["stream_id"],                                                              
            "topic_name": row["topic_name"],                                                            
            "visibility_policy": row["visibility_policy"],                                              
            "last_updated": datetime_to_timestamp(row["last_updated"]),                                 
        }                                                                                               
                                                                                                        
        if include_stream_name:                                                                         
            user_topic_dict["stream__name"] = row["stream__name"]                                       
                                                                                                        
        result.append(user_topic_dict)                                                                  
```

And that `include_stream_name` parameter is only used in one function `get_topic_mutes`, which immediately converts the result into a legacy tuple format that isn't an object at all. So we should delete this line.